avocado/utils/multipath.py

```diff
@@ -30,8 +31,7 @@ def get_svc_name():
     """
     if distro.detect().name == 'Ubuntu':
         return "multipath-tools"
-    else:
-        return "multipathd"
+    return "multipathd"
```

Comment:
This is a minor change, but it's unaccounted for in the commit message.  Also, having individual changes in separate commits is favored.